KEYS

```diff
@@ -132,3 +132,181 @@ MhfLdHrk689ty11m+5HlrSU7O1I1wZkt/OlYsZmS1yIpD1hEnOuSjAuqm4D3s+YI
 B4WM8AJSCwl8WlZrRA==
 =wft0
 -----END PGP PUBLIC KEY BLOCK-----
+
+pub   rsa4096 2020-08-06 [SC]
+      94BD178077672157652826758E44A82497382298
+uid             [ 绝对 ] Shen Yi (CODE SIGNING KEY\) <shenyi@apache.org>
```

Comment:
Seems unexpected Chinese character here.

Reply:
I'm not sure if this can be changed

Reply:
It can be used to verify the signature successfully now, so let's keep it unchanged. Just noticed and mentioned it a bit.